amnestywebsite/humanity-theme
# Index Template: Remove PHP version and update HTML version

Ref: https://github.com/amnestywebsite/humanity-theme/issues/346. Part of https://github.com/amnestywebsite/humanity-theme/issues/246

This removes the index.php template and updates the index.html template to include a `main` container. This should also fix an issue with a blank Editor loading on the initial `/wp-admin/site-editor.php` page.

**Steps to test**:
1. Ensure the correct content is loaded on the front end of your test site on various pages and posts.
2. In wp-admin, go to Appearance -> Editor and ensure that the index template is rendered successfully, rather than a blank Editor.

| site-editor.php Before | site-editor.php After |
| ---- | ---- |
| <img src="https://github.com/user-attachments/assets/2fa8e3ac-a1f7-4997-ac41-a5a176072a2e" width="400" /> | <img src="https://github.com/user-attachments/assets/723c0521-795e-47d3-85d5-273d683250f2" width="400" /> |

**Considerations**:
- Accessibility
  - Regions (semantic HTML tags, e.g. `<section>`; [landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#landmark_roles); etc.)
  - Screen reader compatibility, labels ([`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby), etc.)
  - Keyboard navigability
- Localisation
- RTL
